src/Token/NodeImageBuilder.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\helfi_platform_config\Token;
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\file\FileInterface;
+use Drupal\helfi_api_base\Token\OGImageBuilderInterface;
+use Drupal\media\MediaInterface;
+use Drupal\node\NodeInterface;
+
+/**
+ * OG image for nodes.
+ */
+class NodeImageBuilder implements OGImageBuilderInterface {
```

Comment:
I think we should move this to `helfi_platform_config_base` module because this adds dependency to `helfi_image_styles` module